### ISSUE TITLE
Can't select Stars in the dropdown

### ISSUE THREAD

**Issue by jyng** [urbit/star.market#46]
I can't seem to interact with the dropdown?

![image](https://user-images.githubusercontent.com/9557907/143071150-7c05d6ec-6e03-4b90-aee6-51e2063f726c.png)

I'm running on: 
Chrome Version 95.0.4638.69 (Official Build) (x86_64)
macOS Big Sur 11.5.2

**Comment by willbach** [comment]
it's disabled because you don't have any stars in that account

**Comment by jyng** [comment]
Oh, does the 3 on the left stand for the "Star Market Pool"?

**Comment by willbach** [comment]
yes that's right. I'll update the disabled state copy to "You have no eligible stars"

**Comment by jyng** [comment]
Oh, I logged with Master Ticket, but Master Ticket input is character limited, so I logged into the "wrong" wallet. My Master Ticket is longer than 4 words — that's usually the case if people are logging in with their "Star wallets".

**Comment by willbach** [comment]
ah ok, I'll update that as well then

**Comment by willbach** [comment]
How long is your master ticket?

**Comment by jyng** [comment]
I believe it's 8
